ios/FluentUI/Table View/TableViewHeaderFooterView.swift

```diff
@@ -21,15 +21,14 @@ public protocol TableViewHeaderFooterViewDelegate: AnyObject {
 @available(*, deprecated, renamed: "TableViewHeaderFooterView")
 public typealias MSTableViewHeaderFooterView = TableViewHeaderFooterView
 
-/**
- `TableViewHeaderFooterView` is used to present a section header or footer with a `title` and an optional accessory button.
+/// `TableViewHeaderFooterView` is used to present a section header or footer with a `title` and an optional accessory button.
+///
+/// Set the `TableViewHeaderFooterView.Style` of the view to specify its visual style. While the `default` and `headerPrimary` style may be used for headers, the `footer` style, which lays out the `title` near the top of the view, may be used for footers in grouped lists. Use `divider` and `dividerHighlighted` as headers for plain lists.
```

Comment:
nit: can you break this into 2 lines? (maybe put the break after the "style may be used for headers,"?)